### PR TITLE
feat: auth0 ips { check | unblock }

### PR DESCRIPTION
**Pull request by cyx**
This adds the `auth0 ips` command which supports two operations:

- auth0 ips check <ip>
- auth0 ips unblock <ip>

Quick demo shown below: 

[asciicast](https://asciinema.org/a/XA2JqFei1stMeQZgNJI1pljCR)

### CODE REVIEW

File: internal/cli/ips.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+				inputs.IP = args[0]
+			}
+
+			if err := ansi.Waiting(func() error {
+				return cli.api.Anomaly.UnblockIP(inputs.IP)
+			}); err != nil {
+				return fmt.Errorf("An unexpected error occurred: %w", err)
+			}
+
+			cli.renderer.Infof("The IP %s was unblocked", inputs.IP)
```
Comment (Widcket) [inline internal/cli/ips.go:97]: Can we display the standard header first? With the tenant

File: internal/cli/ips.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+			if err := ansi.Waiting(func() error {
+				var err error
+				isBlocked, err = cli.api.Anomaly.CheckIP(inputs.IP)
+				return err
+			}); err != nil {
+				return fmt.Errorf("An unexpected error occurred: %w", err)
+			}
+
+			if isBlocked {
```
Comment (Widcket) [inline internal/cli/ips.go:60]: Same, let's display the header first.

File: internal/cli/ips.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+func checkIPCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		IP string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "check",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Check IP address.",
```
Comment (Widcket) [inline internal/cli/ips.go:40]:
```suggestion
		Short:   "Check IP address",
```

File: internal/cli/ips.go [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+func unblockIPCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		IP string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "unblock",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Unblock IP address.",
```
Comment (Widcket) [inline internal/cli/ips.go:81]:
```suggestion
		Short:   "Unblock IP address",
```

File: internal/cli/ips.go [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+		Long:    "Unblock IP address which is currently blocked.",
```
Comment (Widcket) [inline internal/cli/ips.go:82]:
```suggestion
		Long:    "Unblock an IP address which is currently blocked.",
```

File: internal/cli/ips.go [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+)
+
+var (
+	ipAddress = Argument{
+		Name: "IP",
+		Help: "IP address to check",
```
Comment (Widcket) [inline internal/cli/ips.go:13]:
```suggestion
		Help: "IP address to check.",
```

File: internal/cli/ips.go [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+	}
+)
+
+func ipsCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "ips",
+		Short:   "Manage blocked IP addresses",
+		Long:    "Manage blocked IP addresses.",
+		Aliases: []string{"ip"},
```
Comment (Widcket) [inline internal/cli/ips.go:22]: Do we need this alias, since `ips` is already short?

Comment (cyx) [reply internal/cli/ips.go:22]: Yeah, probably not, i'll remove.

**Review by Widcket — CHANGES_REQUESTED** [review]
Looks good, just a presentation detail and some punctuation fixes.

**Review by Widcket — APPROVED** [review]

**Pull request merged**